ql/src/test/queries/clientpositive/union_flatten_subdir.q

```diff
@@ -0,0 +1,19 @@
+set hive.tez.union.flatten.subdirectories=true;
```

Comment:
this test should be extended to certain scenarios:

1. acid/mm/non-acid
2. direct insert on/off
3. union flattening on/off